src/coordinate_transformations.jl

```diff
@@ -0,0 +1,91 @@
+mutable struct CRS2CRS <: CoordinateTransformations.Transformation
+    rep::Ptr{Cvoid}
+    direction::PJ_DIRECTION
+end
+
+function CRS2CRS(source::String, target::String, direction::PJ_DIRECTION = PJ_FWD; area = C_NULL, normalize = true)
```

Comment:
I don't think `normalize=true` should be the default if it results in units which are not standard.

Reply:
AFAIK, the only thing normalize does is change the axis order, but I might have missed something in the docs.  I got that info [here](https://proj.org/development/reference/functions.html?highlight=normalize_for_visualization#c.proj_normalize_for_visualization).

Reply:
Oh I see. Well I still think that probably shouldn't be the default. The CRS contains metadata specifying the ordering of components (lat,lon) / (lon,lat) / (northing,easting) / (easting,northing), so asking for a specific CRS is asking for that ordering and it should probably be respected.

Granted there's a tension here between analytical work where normalize=false should be preferred, vs plotting where convenience is arguably more important. But I think as a Proj API it should be correct rather than convenient by default :-)